### PR TITLE
Load CARE, CARE Plus 311 data in DW

### PR DESCRIPTION
**Pull request by hunterowens**
This PR creates a single task nightly DAG that loads the CARE and CARE PLUS data.

### CODE REVIEW

File: dags/public-health/care-311-to-postgres.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+def load_to_postgres():
+    """
+    Loads the care, care plus response codes 
+    to postgres as an upsert
+    """
+    dataset_id = 'jvre-2ecm'
+    client = Socrata("data.lacity.org",
+                     "PrfihrlGM6GihbDbDRHqtQ0R4",
+                     username="hunter.owens@lacity.org",
+                     password="e*75BTtTP#Qm*c^N8J7y")
```
Comment (ian-r-rose) [inline dags/public-health/care-311-to-postgres.py]: Maybe amend this commit.

Comment (hunterowens) [reply dags/public-health/care-311-to-postgres.py]: yep, just caught.

File: dags/public-health/care-311-to-postgres.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+    results = []
+    req_count = 0
+    page_size = 2000
+    data = None
+    while data != []:
```
Comment (ian-r-rose) [inline dags/public-health/care-311-to-postgres.py]: Wow, this is not a very nice API!

Comment (ian-r-rose) [reply dags/public-health/care-311-to-postgres.py]: Are there any other falsy values that could be returned by `client.get`?

File: dags/public-health/care-311-to-postgres.py (hunk 1 above)
Comment (ian-r-rose) [reply dags/public-health/care-311-to-postgres.py]: Hmm, still showing in the UI :/

**Pull request closed without merging**